ATACseq_BonusTutorial.ipynb

```diff
@@ -300,10 +299,10 @@
     "<details>\n",
     "  <summary>Click for help</summary>\n",
     "    \n",
-    "  ```  \n",
-    "  dframe = pd.read_csv(\"Tutorial1/QC/multiqc_data/multiqc_fastqc.txt\", sep='\\t')\n",
-    "  display(dframe)\n",
-    "  ```\n",
+    "  \n",
+    "  `dframe = pd.read_csv(\"Tutorial1/QC/multiqc_data/multiqc_fastqc.txt\", sep='\\t')`  \n",
```

Comment:
In the code block lets remove the exclamation point since these are python commands. and maybe add a comment in the Click for help box that says the following "This is a python command so it does not need a exclamation point (!) before it to run unlike the bash commands we have been running so far."

Reply:
Done.